rclpy/rclpy/duration.py

```diff
@@ -76,3 +76,6 @@ def from_msg(cls, msg):
         if not isinstance(msg, builtin_interfaces.msg.Duration):
             raise TypeError('Must pass a builtin_interfaces.msg.Duration object')
         return cls(seconds=msg.sec, nanoseconds=msg.nanosec)
+
+    def get_c_duration(self):
```

Comment:
nit, recommend using `@property` instead of a `get_` method in python.

```python3
@property
def duration_handle(self):
    return self._duration_handle
```

Reply:
I was copying a "c-handle pattern" that I observed here https://github.com/ros2/rclpy/blob/master/rclpy/rclpy/qos.py#L124 - do you think I should change both of these to use properties instead?

Reply:
Rodger, Feel free to ignore this comment and use the same pattern. If the pattern should be changed it doesn't need to block this PR.